server/testutils/jest.setup.ts

```diff
@@ -28,7 +28,7 @@ expect.extend({
   },
   toMatchOpenAPISpec(pactPath) {
     const openAPIUrl =
-      'https://raw.githubusercontent.com/ministryofjustice/hmpps-approved-premises-api/main/src/main/resources/static/codegen/built-cas2-api-spec.yml'
+      'https://raw.githubusercontent.com/ministryofjustice/hmpps-approved-premises-api/CAS2-327-repoint-notes-to-assessments-part-1/src/main/resources/static/codegen/built-cas2-api-spec.yml'
```

Comment:
gentle reminder to drop this commit before merging :)

Reply:
Dropped now!